stats/opentelemetry/client_metrics.go

```diff
@@ -68,6 +73,17 @@ func (h *clientStatsHandler) initializeMetrics() {
 	rm.registerMetrics(metrics, meter)
 }
 
+func (h *clientStatsHandler) initializeTracing() {
+	// Will set no traces to record if either of (TraceProvider, TextMapPropagator)
+	// are nil, logically making this stats handler a no-op.
```

Comment:
it is not making stats handler no-op. Only tracing part is no-op

Reply:
Updated